src/Ng/ElfieFromCatalogCollector.cs

```diff
@@ -136,6 +165,42 @@ public ElfieFromCatalogCollector(Version indexerVersion, Version mergerVersion,
         }
 
         /// <summary>
+        /// Gets the list of fake packages which represent assemblies on the local machine.
+        /// </summary>
+        /// <param name="assemblyPackageDirectory">The directory which contain the text files representing the local packages.</param>
+        /// <returns>Returns faked package registration objects which contain the information for the local packages.</returns>
+        IList<Tuple<RegistrationIndexPackage, long>> GetLocalAssemblyPackages(string assemblyPackageDirectory)
```

Comment:
I will add an entry in the onenote about how to create these files.

The files are created manually. It's basically:
dir /s /b directorywithassemblies > PackageName.txt